.travis.yml

```diff
@@ -28,22 +28,32 @@ jobs:
       compiler: gcc
       name: "gcc"
       if: NOT type = cron
-      before_install: .travis/before_install.sh
+      env:
+        - PATH="$HOME/miniconda/bin:$PATH"
```

Comment:
If this is now needed everywhere could you move it to the `global` section above (same for all other stages).